import_cambridge_lands.py

```diff
@@ -26,7 +26,9 @@ def __init__(
             self.lr_api_url = self.lr_api_url + '/'
 
     def get_lr_data(self, uprn):
-        url = '{0}{1}'.format(self.lr_api_url, uprn)
+        # some gazetteer database applications will pad UPRNs that are less
+        # than 12 digits with zeros BUT the HMRC API needs you to remove them
+        url = '{0}{1}'.format(self.lr_api_url, int(uprn))
```

Comment:
I'm not 100% sure if we should do this... I remember @rossjones told me to explicitly add the 0 padding otherwise UPRNs wouldn't match, but I may remember wrongly...should we ask him maybe?